Flow.Launcher/Languages/en.xaml

```diff
@@ -3,7 +3,8 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:system="clr-namespace:System;assembly=mscorlib">
     <!--  MainWindow  -->
-    <system:String x:Key="registerHotkeyFailed">Failed to register hotkey: {0}</system:String>
+    <system:String x:Key="registerHotkeyFailed">Failed to register hotkey "{0}". The hotkey may be in use by another program. Change to a different hotkey, or exit another program.</system:String>
+    <system:String x:Key="registerHotkeyFailedTitle">Flow Launcher</system:String>
```

Comment:
Consider changing "registerHotkeyFailedTitle" to something like "MessageBoxTitle" so we can reuse it for other messageboxes?

Reply:
fixed to "MessageBoxTitle".